docs/writing-policies/rust/01-intro-rust.md

```diff
@@ -1,47 +1,38 @@
 ---
-sidebar_label: "Introduction to Rust"
-title: ""
+title: "Rust"
+description: An introduction to writing Kubewarden policies with Rust.
+keywords: [kubewarden, kubernetes, writing policies, introduction]
+doc-type: [tutorial]
+doc-topic: [writing-policies, rust, introduction]
 ---
 
-# Rust
+[Rust](https://www.rust-lang.org/) is a mature programming language that can generate WebAssembly modules.
```

Comment:
```suggestion
[Rust](https://www.rust-lang.org/) is the most mature programming language that can generate WebAssembly modules.
```

I would keep the pointer here, it's still true, and seems that will be for the foreseeable. It also explains why we favour rust for our policies.

Reply:
I agree with what @viccuad said

Reply:
Thanks, and fixed.